pkg/autoupdate/tuf/library_manager.go

```diff
@@ -0,0 +1,349 @@
+package tuf
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/Masterminds/semver"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/fsutil"
+	"github.com/kolide/kit/version"
+	"github.com/kolide/launcher/pkg/autoupdate"
+	"github.com/theupdateframework/go-tuf/data"
+	"github.com/theupdateframework/go-tuf/util"
```

Comment:
mild preference for something like 
```suggestion
	tufutil "github.com/theupdateframework/go-tuf/util"
```

I spent a minute trying to figure out what util was when I read it in the code. 😆